docs/docs/external/getting-started/test-network/ledger-initialization.md

```diff
@@ -148,6 +150,17 @@ Finally, to prepare both `network1` and `network2` for interoperation, run:
 ./scripts/initAsset.sh
 ```
 
+### Initializing the Corda Networks
+
+Corda Networks needs to be initialized with assets for asset-exchange to be performed:
```

Comment:
"Cora Networks needs" --> "Either or both Corda networks need"

Reply:
For the purpose of demo, we have not shared any instruction to start only one network. So do you think `Either one` is appropriate?

Reply:
Ok, then just say "Corda Network". I looked at the plural and thought you were providing instructions for both networks.